README.md

```diff
@@ -6,61 +6,255 @@ Cookiecutter template for new CDP deployments.
+[Cookiecutter Repo Generation](#cookiecutter-repo-generation)
+
+### Cookiecutter Parameters
+
+| Parameter                      | Description                                                                                                                | Example 1                             | Example 2                                |
+| ------------------------------ | -------------------------------------------------------------------------------------------------------------------------- | ------------------------------------- | ---------------------------------------- |
+| municipality                   | The name of the municipality (town, city, county, etc.) that this CDP Instance will store data for.                        | Seattle                               | King County                              |
+| municipality_slug              | The name of the municipality cleaned for use in infrastructure and certain parts of repository naming.                     | seattle                               | king-county                              |
+| python_municipality_slug       | The name of the municipality cleaned for use in specifically Python parts of the application.                              | seattle                               | king_county                              |
+| maintainer_or_org_full_name    | The full name of the primary maintainer or organization that will be managing this instance deployment.                    | Jackson Maxfield Brown                | CouncilDataProject                       |
+| hosting_github_username_or_org | The GitHub username or organization that will host this instance's repository. (Used in the web application's domain name) | JacksonMaxfield                       | CouncilDataProject                       |
+| hosting_github_repo_name       | A specific name to give to the repository. (Used in the web application's full address)                                    | cdp-seattle                           | king-county                              |
+| hosting_github_url             | From the provided information, the expected URL of the web application.                                                    | jacksonmaxfield.github.io/cdp-seattle | councildataproject.github.io/king-county |
+
+### Things to Know
+
+Much of CouncilDataProject processing and resource management can be handled for
+free and purely on GitHub. However we do rely on a select few resources outside
+of GitHub to manage all services and applications.
+
+The only service that will require a billing account to manage payment for
+resources used, is [Google Cloud](#google-cloud).
+Google Cloud will manage all databases, file storage, and (if needed)
+[speech-to-text](#speech-to-text) for transcription. You can see more about the
+average monthly cost of running a CDP Instance in [Cost](#cost).
+
+[Pulumi](#pulumi) is a service to manage and track infrastructure deployment state.
```

Comment:
It looks like there isn't a Pulumi heading in this README, so this link doesn't go to anything